engine/src/main/java/org/hibernate/search/bridge/builtin/impl/NullEncodingTwoWayFieldBridge.java

```diff
@@ -58,6 +59,7 @@ public String objectToString(Object object) {
 		}
 	}
 
+	@Override
 	public TwoWayFieldBridge unwrap() {
```

Comment:
I did not consider it, you're right. The `unwrap()` method in `NullEncodingTwoWayFieldBridge` was already there and I just generalized its use. Maybe following the `unwrap(Class<?>)` pattern would be better.

I'll see what I can do, but it probably won't look perfect: we actually have two ways of unwrapping: one which goes through container bridges, the other that doesn't.